Reading List/Reading List/BookTableViewCellDelegate.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  BookTableViewCellDelegate.swift
+//  Reading List
+//
+//  Created by James Sedlacek on 6/5/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+protocol BookTableViewCellDelegate {
```

Comment:
I would suggest constraining all delegates to AnyObject and making them weak variables to avoid retain cycles (which you will learn about later in the course)

``` swift
protocol BookTableViewCellDelegate: AnyObject
```